src/sql/media/icons/common-icons.css

```diff
@@ -546,7 +548,7 @@ Includes non-masked style declarations. */
 }
 
 .book.codicon {
-
+	-webkit-mask-image: url("book_image.svg");
```

Comment:
You have this twice now?

Reply:
@Charles-Gagnon @alanrenmsft Let's get this focus fix in and settle on colors next.

Reply:
Fixed